src/Components/Alert/Alert_Controller.php

```diff
@@ -27,10 +27,21 @@ public function render(): void {
 
 		echo $this->view->render( self::VIEW, [
 			'dto'             => $alert,
+			'alert_classes'   => $this->get_alert_classes( $alert ),
 			'link_attributes' => $this->get_link_attributes( $alert ),
 		] );
 	}
 
+	protected function get_alert_classes( Alert_Dto $alert ): string {
+		$classes = [ 'tribe-alerts' ];
+		if ( $alert->color_class ) {
+			$classes[] = 'tribe_alerts--colored';
```

Comment:
```suggestion
			$classes[] = 'tribe-alerts__has-color-theme';
```
Match our css naming style